packages/query/src/index.ts

```diff
@@ -844,7 +848,7 @@ const generateQueryImplementation = ({
           )
             return param.destructured;
           return param.name === 'params'
-            ? `{ ${queryParam}: pageParam, ...params }`
+            ? `{...params, ${queryParam}: pageParam }`
```

Comment:
My question is the same as @anymaniax asked in the other ticket does this change break backwards compatibility?

Reply:
I'm thinking of doing this with another pull request.